python/understack-workflows/understack_workflows/sync_interfaces.py

```diff
@@ -0,0 +1,114 @@
+import os
```

Comment:
This whole file seems to duplicate what's in main/sync_interfaces.py any reason not to move that?